lstchain/reco/utils.py

```diff
@@ -119,10 +132,11 @@ def compute_theta2(data, source_position, conversion_factor=2.0):
     -------
     Array with `theta2` values
     """
-    reco_src_x = np.array(data['reco_src_x']) * u.m
-    reco_src_y = np.array(data['reco_src_y']) * u.m
-    return conversion_factor**2 * ((source_position[0] - reco_src_x)**2 +
-                                   (source_position[1] - reco_src_y)**2)
+    reco_src_x = np.array(data["reco_src_x"]) * u.m
+    reco_src_y = np.array(data["reco_src_y"]) * u.m
+    return conversion_factor ** 2 * (
+        (source_position[0] - reco_src_x) ** 2 + (source_position[1] - reco_src_y) ** 2
+    )
 
 
 def compute_alpha(data):
```

Comment:
we already have a function to compute `alpha`:
https://github.com/cta-observatory/cta-lstchain/blob/7fb83894c25d26360906ddce04bb89a6af49471c/lstchain/reco/dl1_to_dl2.py#L501

I would avoid code duplication

Reply:
This is not part of this PR. But I can remove it.